styles.css

```diff
@@ -389,4 +406,309 @@ textarea {
   color: #9490ff;
   font-size: 12px;
   line-height: 24px;
+  font-family: "Roboto", sans-serif;
+}
+
+.arrow-1 {
+  display: none;
+}
+
+/*
+  DESKTOP VERSION
+*/
+
+@media only screen and (max-width: 1100px) and (min-width: 992px) {
+  html,
+  body {
+    height: 100%;
+    margin: 0;
+  }
+
+  body {
+    background: url(/images/two-circles-header.png);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-position: center;
+    background-color: #191765;
+  }
+
+  .header-box {
+    width: 85%;
+    margin: 50px auto;
+  }
+
+  main {
+    background: url(/images/Group-elipse.png);
+    background-repeat: no-repeat;
+    background-position-y: 83%;
+    background-size: 100% 55%;
+    width: 100%;
+    margin: 0;
+  }
+
+  .nav-container {
+    display: flex;
+    flex-direction: row;
+    gap: 25px;
+  }
+
+  .nav-container a {
+    display: flex;
+    justify-content: space-between;
+    color: #fff;
+    font-size: 15px;
+    line-height: 20px;
+    font-weight: 500;
+    font-family: "Roboto", sans-serif;
+    text-align: center;
+    text-decoration: none;
+  }
+
+  .nav-container img {
+    display: none;
+  }
+
+  .headline-wrapper > * {
+    width: 45%;
+    margin: 0 auto;
+  }
+
+  .name-info {
+    margin: 50px auto 50px;
+  }
+
+  .name-info h1 {
+    font-size: 50px;
+    line-height: 105.47px;
+    font-weight: 700;
+    margin: 0;
+  }
+
+  .name-info h2 {
+    font-size: 40px;
+    line-height: 50px;
+    font-weight: 500;
+    margin: 0;
+  }
+
+  #intro {
+    line-height: 32px;
+  }
+
+  .skills-item {
+    margin: 0;
+  }
+
+  .skills-item > * {
+    flex: 1;
+    margin-left: 45px;
+  }
+
+  .skills-item li {
+    font-size: 18px;
+    line-height: 30px;
+    font-weight: 700;
+  }
+
+  .arrow-1 {
+    display: flex;
+  }
+
+  .support_items {
+    width: 80%;
+    flex-flow: row;
+    justify-content: space-between;
+    align-items: center;
+    margin: 100px auto 200px;
+  }
+
+  .work-direction {
+    padding: 0;
+  }
+
+  .social-media > * {
+    margin: 0;
+  }
+
+  .skills-item ul > * {
+    font-size: 14px;
+    line-height: 28px;
+    font-weight: 400;
+    font-family: "Roboto", sans-serif;
+  }
+
+  .skills-item > *:nth-child(1) {
+    margin-left: 10px;
+  }
+
+  .work-direction span {
+    font-size: 20px;
+    font-weight: 500;
+    color: #b3bac5;
+  }
+
+  .work-direction img {
+    color: #c4c4c4;
+  }
+
+  .work {
+    width: 90%;
+    margin: 0 auto;
+    margin-bottom: 140px;
+  }
+
+  #work-title {
+    margin-bottom: 90px;
+  }
+
+  #break {
+    display: inline;
+  }
+
+  .all_projects {
+    grid-gap: 90px;
+  }
+
+  .project {
+    flex-flow: row;
+    width: 100%;
+    gap: 4%;
+    margin: 0;
+  }
```

Comment:
- [ ] Please make sure that your design follows the original Figma template as much as possible. I'm referring to the Figma template here:

Figma template:
![Captura de pantalla 2022-01-12 180848](https://user-images.githubusercontent.com/59100477/149188052-cb8b73d4-5118-456e-ab92-d8bbb1adeaaf.jpg)

And your design it's actually only showing a big image and the text below, but you should place the text next to the image:
![Captura de pantalla 2022-01-12 181002](https://user-images.githubusercontent.com/59100477/149188191-58c68dde-5616-44b2-b2ce-42980c30a33b.jpg)

So please make the necessary changes.

- [ ] [OPTIONAL] I would suggest making the images a bit more responsive, because right now they are taking too much space, and even when zooming out, they keep their size, and would be nice if you figure out a way to make then behavior a bit better. (Remember that all OPTIONAL requirements are not mandatory, and are not enough no invalidate your project, but they are a good chance to improve!)